oauth2_provider/forms.py

```diff
@@ -80,7 +80,7 @@ def clean(self):
             except User.DoesNotExist:
                 user = None
 
-        if user is None or not user.is_active:
```

Comment:
Good. Would you ensure that a task is added to restore the `is_active` check here, please? Otherwise, we don't have a method of preventing inactive users from authenticating.